lib/charms/layer/kubernetes_master.py

```diff
@@ -21,6 +21,8 @@
 
 AUTH_BACKUP_EXT = 'pre-secrets'
 AUTH_BASIC_FILE = '/root/cdk/basic_auth.csv'
+AUTH_SECRET_NS = 'kube-system'
+AUTH_SECRET_SUFFIX = 'token-auth'
```

Comment:
I'm happy to bike shed this suffix, but since we're using `juju.is/token-auth` as the secret type, I figured i'd keep it close to that.